interface-definitions/protocols_segment-routing.xml.in

```diff
@@ -126,6 +126,25 @@
                     </properties>
                     <defaultValue>24</defaultValue>
                   </leafNode>
+                  <leafNode name="format">
+                    <properties>
+                      <help>Configure SRv6 SID format</help>
```

Comment:
```suggestion
                      <help>SRv6 SID format</help>
```

We generally avoid verbs in help strings. Configuring something is exactly what configuration commands do. :)